### PR TITLE
Serve user statistics via javascript

### PR DESCRIPTION
**Pull request by sethvargo**
**Release Note**

```release-note
Serve user statistics via javascript
```

/assign @whaught

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -7,11 +7,6 @@
 - [API Methods](#api-methods)
   - [`/api/verify`](#apiverify)
   - [`/api/certificate`](#apicertificate)
-- [Admin APIs](#admin-apis)
-  - [`/api/issue`](#apiissue)
```
Comment (whaught) [inline docs/api.md:11]: we should probably leave a link to /api/issue in the TOC

File: pkg/controller/stats/stats.go [hunk 2]
```diff
@@ -54,11 +54,16 @@ func New(ctx context.Context, cacher cache.Cacher, db *database.Database, h rend
 }
 
 // authorizeFromContext attempts to pull authorization from the context. It
-// returns false if authorization failed.
-func authorizeFromContext(ctx context.Context) (*database.Realm, bool) {
+// returns false if authorization failed. The authorization must have one of the
+// provided permissions. to succeed
```
Comment (whaught) [inline pkg/controller/stats/stats.go:58]:
```suggestion
// provided permissions to succeed.
```

File: cmd/server/assets/users/show.html [hunk 3]
```diff
@@ -163,26 +153,28 @@ <h5 class="modal-title">Codes issued by {{$user.Name}}</h5>
           dataTable.addRow([utcDate(row.date), row.data.codes_issued]);
         });
 
+        let dateFormatter = new google.visualization.DateFormat({
+          pattern: 'MMM dd',
```
Comment (whaught) [inline cmd/server/assets/users/show.html:157]: I'm not sure the pattern syntax, but I've never seen a three-character `MMM` for month?

Comment (sethvargo) [reply cmd/server/assets/users/show.html:157]: It's "Nov" "Dec" according to their docs

**Review by whaught — APPROVED** [review]

File: cmd/server/assets/users/show.html (hunk 3 above)
Comment (whaught) [reply cmd/server/assets/users/show.html:157]: ah

**Review by whaught — APPROVED** [review]

**Pull request merged**